![alt text](images/7.1-15.1.png)

![alt text](images/7.1-15.2.png) 

In [11]:
from helper import * 


samples = load_sample_data(7,1,15,display=True)
xBar = sample_mean(samples)
s = sample_variance(samples) **.5

tInterval = scipy.stats.t.ppf(.99, len(samples)-1)
lowerBound = xBar - tInterval*(s/(len(samples) ** .5))

Loaded E7_1-15.txt sucessfully.
[28.8 24.4 30.1 25.6 26.4 23.9 22.1 22.5 27.6 28.1 20.8 27.7 24.4 25.1
 24.6 26.3 28.2 22.2 26.3 24.4]
25.475 2.493544296357042 24.059051583093837 2.539483190622288


## Lowerbound $= P( \bar{X} - t_{1-\alpha}(\frac{s}{\sqrt{n}}) \leqq \mu) = 1 - \alpha$ <br/>*As proven in previous problem* 
# $ t_{1-.99} = t_{.01}= 2.539483190622288 \approx 2.539 \\ \bar{X} - t_{1-\alpha}(\frac{s}{\sqrt{n}}) = 25.475 - 2.539(\frac{2.493544296357042}{\sqrt{20}}) = 24.059051583093837 \approx 24.05905 \\  [24.05905, \infty)$